pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1641,6 +1641,7 @@ const (
 //
 // - `none`: doing nothing.
 // - `delete-pod`: delete pod and remove the evict-leader scheduler from PD.
+// - `delete-pod-and-flush-log-backup`: delete pod, remove the evict-leader scheduler from PD and flush the running log backup task.
```

Comment:
where is the code change?

Reply:
In fact this is the initial design, however it seems useless so I removed it and made all `delete-pod` restarting flushes the task.